django_simple_bulma/utils.py

```diff
@@ -0,0 +1,70 @@
+"""Django Simple Bulma utilities. Ultimately helps insure DRY code."""
```

Comment:
```suggestion
"""Django Simple Bulma utilities. Ultimately helps ensure DRY code."""
```

To insure it against water damage, maybe? We don't want the code getting WET.